lib/cdo/graphics/certificate_image.rb

```diff
@@ -90,7 +88,7 @@ def create_workshop_certificate_image(image_path, fields)
 
 # Prepare the given string for using in Image Magick.
 def escape_image_magick_string(string)
-  string = string.force_8859_to_utf8
+  string = string.dup.force_8859_to_utf8
```

Comment:
Why remove the `dup`?

Reply:
I'm actually adding `.dup`. It was receiving a "frozen" string and I guess `force_8859_to_utf8` modifies the object and should have a bang in the name. I'm looking for documentation though.

Reply:
🤦‍♀

Reply:
Oh it's a method we created. Yes it does modify the existing string and should have a bang.